swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/model/WorkConfiguration.kt

```diff
@@ -40,3 +41,8 @@ class EmptyNotificationConfiguration : NotificationConfiguration(
   types = mutableListOf(),
   optOutBaseUrl = ""
 )
+
+data class SoftDelete(
+  var enabled: Boolean = false, // controls if soft deleting is enabled for this work configuration
+  var waitingPeriodDays: Int = 2 // How many days swabbie will wait after soft deleting a resource before deleting it
```

Comment:
Prefer this documentation form for data classes & constructors, as it'll actually get picked up by doc generators:

```
/**
 * @param enabled controls if soft deleting is...
 */
data class SoftDelete( ... )
```